src/react/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useMemo, useState } from 'react';
+
+export type MediaQueryType = 'only' | 'up' | 'down' | 'between';
+export type MediaQueryBreakpoint = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+const breakpoints: Record<string, number> = {
```

Comment:
Ik denk wel dat het een gemis is dat je zelf de breakpoints niet aan kan passen. Je zou een parameter kunnen toevoegen waar je deze array kan overschrijven. Nadeel is wel dat je dat dan bij elke hook moet doen. 

Of voor de uitzonderingen kunnen kiezen en het mogelijk maken dat je ook een number als breakpoint kan opgeven.

Reply:
@martinbroos ,
Het toevoegen van een custom breakpoint als number is niet zo'n slecht idee.
En wat dacht je van een ENV variabelen waar je de breakpoints kan zetten op een bepaalde manier bijv:
`sm:600,md:960` etc.
Dus default is dit, anders ENV file die hij inleest en met process.env.MEDIA_QUERY_BREAKPOINTS ophaalt?